tests: Add retry mechanism when checking certificate renewal

alexandre-allard opened the pull request:
Lower the wait time, but retries 3 times before exiting in error.
In best cases the test is faster, otherwise we wait a little more than before.

alexandre-allard committed:
tests: Add retry mechanism when checking certificate renewal

Lower the wait time, but retries 3 times before
exiting in error.
In best cases the test is faster, otherwise we
wait a little more than before.

bert-e commented:
# Hello alexandre-allard-scality,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbertran

* @chengyanjin

* @slaperche-scality

* @thomasdanan

* @lucieleonard

* @gdemonet

* @teddyandrieux

* @nootal

* @monpote

* @tcarmet

* @sayf-eddine-scality

* @cmonfort

* @nicolast

* @carlito-scality

* @alexandre-allard-scality

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/raise-certificate-renewal-wait-time` into
`development/2.7`
* [w/2.8/bugfix/raise-certificate-renewal-wait-time](https://github.com/scality/metalk8s/compare/development/2.8...w/2.8/bugfix/raise-certificate-renewal-wait-time) will be merged into `development/2.8` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbertran

* @chengyanjin

* @slaperche-scality

* @thomasdanan

* @lucieleonard

* @gdemonet

* @teddyandrieux

* @nootal

* @monpote

* @tcarmet

* @sayf-eddine-scality

* @cmonfort

* @nicolast

* @carlito-scality

* @alexandre-allard-scality

alexandre-allard committed:
eve: Wait for pods stabilization after certificate renewal

bert-e commented:
# History mismatch



Merge commit #be913928199e92ab77c09e7716a8c0d711bccd5b on the integration branch
`w/2.8/bugfix/raise-certificate-renewal-wait-time` is merging a branch which is neither the current
branch `bugfix/raise-certificate-renewal-wait-time` nor the development branch
`development/2.8`.

It is likely due to a rebase of the branch `bugfix/raise-certificate-renewal-wait-time` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

alexandre-allard commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/raise-certificate-renewal-wait-time` into
`development/2.7`
* [w/2.8/bugfix/raise-certificate-renewal-wait-time](https://github.com/scality/metalk8s/compare/development/2.8...w/2.8/bugfix/raise-certificate-renewal-wait-time) will be merged into `development/2.8` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbertran

* @chengyanjin

* @slaperche-scality

* @thomasdanan

* @lucieleonard

* @gdemonet

* @teddyandrieux

* @nootal

* @monpote

* @tcarmet

* @sayf-eddine-scality

* @cmonfort

* @nicolast

* @carlito-scality

* @alexandre-allard-scality

alexandre-allard committed:
tests: Reset beacon configuration at end of tests

Avoid the beacon to keep sending event for certificates
renewal which loads the salt minion and master and
make the platform really slow and further tests flaky.

alexandre-allard commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbertran

* @chengyanjin

* @slaperche-scality

* @thomasdanan

* @lucieleonard

* @gdemonet

* @teddyandrieux

* @nootal

* @monpote

* @tcarmet

* @sayf-eddine-scality

* @cmonfort

* @nicolast

* @carlito-scality

* @alexandre-allard-scality










*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/267a1ace2169e2321abce2527e64c4aea9954707) did not succeed in branch bugfix/raise-certificate-renewal-wait-time.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches have **NOT** changed:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`



Please check the status of the associated issue None.

Goodbye alexandre-allard-scality.
